paasta_tools/contrib/service_shard_update.py

```diff
@@ -118,7 +158,13 @@ def main(args):
                     if args.shard_name not in kube_file.keys():
                         kube_file[args.shard_name] = {
                             "deploy_group": f"{deploy_prefix}.{args.shard_name}",
-                            "instances": args.instance_count,
+                            "min_instances": args.min_instance_count,
+                            "max_instances": args.prod_max_instance_count
+                            if deploy_prefix == "prod"
+                            else args.non_prod_max_instance_count,
```

Comment:
yikes did the black autoformatting do this?  :-/

Reply:
oh it did